README.md

```diff
@@ -83,15 +74,29 @@ git clone https://github.com/DocDrydenn/kiwix-zim.git
 ```
 
 UPDATE: If you decide not to install via a git clone, you can still use this script, however, it will just skip the update check and continue on.
+NOTE: if you are not tracking the `main` branch, the update check will be skipped. So if you do not want to get updates, but like git, just track the commit of your choosing.
 
 ## Usage
 
 ```text
-    Usage: ./kiwix-zim.sh <h|d> /full/path/
-
-    /full/path/       Full path to ZIM directory
-
-    -d or d           Dry-Run Override
-
-    -h or h           Show this usage and exit
+  Usage: ./kiwix-zim.sh <options> /full/path/
+  
+      /full/path/                Full path to ZIM directory
+  
+  Options:
+      -c, --calculate-checksum   Verifies that the downloaded files were not corrupted, but can take a while for large downloads.
+      -f, --verify-library       Verifies that the entire library has the correct checksums as found online.
+                                 For this reason, a file `library.sha256` will be left in your library for running sha256sum manually
```

Comment:
```suggestion
                                 N.B. a file `library.sha256` will be left in your library regardless of this option, so that the -f option will run offline if requesed
```
If you accept this edit, you would also need to make it in the script's option list as well.